src/firejail/dhcp.c

```diff
@@ -100,7 +101,7 @@ static pid_t dhcp_read_pidfile(const Dhclient *client) {
 			long pid;
 			if (fscanf(pidfile, "%ld", &pid) == 1) {
 				char *pidname = pid_proc_comm((pid_t) pid);
-				if (pidname && strcmp(pidname, "dhclient") == 0)
+				if (pidname && strcmp(pidname, dhclient_path))
```

Comment:
you don't want equal strings any more?

Reply:
Whoops, good catch!

Reply:
I don't know why I was fumbling around with this. `pid_proc_comm` returns `"dhclient"` even if it was started as `/sbin/dhclient`, so the previous check was actually correct.